pkg/controllers/provisioning/scheduling/instance_selection_test.go

```diff
@@ -528,8 +529,45 @@ var _ = Describe("Instance Type Selection", func() {
 						v1.ResourceCPU:    resource.MustParse(fmt.Sprintf("%0.1f", cpu)),
 						v1.ResourceMemory: resource.MustParse(fmt.Sprintf("%0.1fGi", mem)),
 					}}}
+				opts1 := test.PodOptions{
```

Comment:
Can we be more explicit with the naming or with a comment around what `opts1` and `opts2` are? I think in general we should be striving to have naming or comments that make it so that someone doesn't have to read the detail to understand what's in the options. Right now, the name has no meaning so I have to go and read the detail to see what the options contain.